src/serum-tokenomics.yaml

```diff
@@ -1,19 +1,349 @@
-uuid: 45a31914-5cd7-4875-b099-c28dc55eed0d
+uuid: 91fa6e9f-ac9d-4135-8b84-aa05ba673389
 key: serum-tokenomics
-name: Build on Serum's Dex
-content: Explains how devs can build on top of Project Serum's DEX
+name: Serum Tokenomics
+content: Explains SRM tokens
 guideType: onboarding
-created: "2022-08-02T04:15:13.911Z"
+created: "2022-08-21T04:15:13.911Z"
 categories:
   - developer
   - engineering
-thumbnail: ipfs://QmNd8S5Z9T8cWv95UUjn5fYR1wU8nLWjeHzXoRC5oVcVe8
+thumbnail: https://d31h13bdjwgzxs.cloudfront.net/guides/eth/aave/AAVE_Portals.png
 publishStatus: Live
 socialShareImage:
 showIncorrectOnCompletion: true
 steps:
-  - uuid: 1f2a1def-610b-4c47-ac4a-162aa055103e
-    name: Developer Guide
+  - uuid: 46c146d5-ff73-4099-bb13-b0161fe90152
+    name: Introduction
     content: |
-      Hard core developer guide which explains what APIs to call to integrate with Serum DEX
+      ## What is Serum?
+      The serum is a protocol designed to supercharge decentralized finance applications. It's built on Solana and completely permissionless, which means anyone can use it to create robust financial markets composites. 
```

Comment:
WE can just write one paragraph about serum in the first step. We want the guide to be specific. 

Because if we write the introductory step to every guide, the end user will feel that all the guides are same